kubernetes-manifests/adservice.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: server
-        image: adservice
+        image: gcr.io/stackdriver-sandbox-230822/sandbox/adservice:latest
```

Comment:
Hardcoding the image (even if we use latest tag here) means that if we want to test things locally, we'll have to update the image path manually in all the microservices yaml files. Why not using skaffold profiles feature where we could just specify the image prefix once (--default-repo)  and not in 10 places?

Reply:
This is actually the way skaffold is intended to work ([source](https://skaffold.dev/docs/environment/image-registries/)). It will rewrite gcr.io/stackdriver-sandbox-230822/sandbox/adservice:latest to $DEFAULT_REPO/gcr.io/stackdriver-sandbox-230822/sandbox/adservice:$hash

The re-written path is a bit ugly, but I think we're better off following skaffold's best practices instead of writing custom scripts to do rewriting ourselves